### PR TITLE
Added more Fan actions and Routines

### PR DESCRIPTION
**Pull request by samuel-verdonck**
Fixes # .

Changes proposed in this pull request: 

* Added actions
  * Basic fan control (switched-fan)
  * Ventilation control (fan)
    * 0 = Low 
    * 1-49 = Medium 
    * 50-99 = High 
    * 100 = Boost
* Added routines
  * All-Off
  * Presence Simulation
  * Alarms
  * Scenes (comfort)
  * Peak Mode
  * Solar Mode

### CODE REVIEW

**Comment by samuel-verdonck** [comment]
@rubencallewaert, Would it be possible to review and let me know what you think?

File: src/nhc2-platform.ts [hunk 1]
```diff
@@ -121,31 +122,72 @@ class NHC2Platform implements DynamicPlatformPlugin {
...
         service: this.Service.Switch,
-        handlers: [this.addTriggerCharacteristic],
+        handlers: [this.addTriggerCharacteristic]
       },
-      sunblind : {
-        service : this.Service.WindowCovering,
-        handlers : [this.addPositionChangeCharacteristic]
+      'switched-generic': {
+        service: this.Service.Switch,
+        handlers: [this.addStatusChangeCharacteristic],
```
Comment (rubencallewaert) [inline src/nhc2-platform.ts:129]: Sometimes I see trailing commas, and sometimes I don't. Could you make a change to make it consistent?

**Review by rubencallewaert — CHANGES_REQUESTED** [review]
Thanks for this merge request, looks good to me besides the tiny comment.

File: src/nhc2-platform.ts (hunk 1 above)
Comment (samuel-verdonck) [reply src/nhc2-platform.ts]: Fixed

**Pull request merged**